Adding extension.neon to .gitattributes breaks other tools

<!--
    🛑 DON'T REMOVE ME.
    This issue template apply to all
      - bug reports,
      - feature proposals,
      - and documentation requests

    Having all those informations will allow us to know exactly
    what you expect and answer you faster and precisely (answer
    that matches your Carbon version, PHP version and usage).
    
    Note: Comments between <!- - and - -> won't appear in the final
    issue (See [Preview] tab).
-->
Hello,

I encountered an issue with the following code:
```php
phpstan analyze
```

Carbon version: 2.51.0

PHP version: 8.0.6

<!--
    Run the command `composer show nesbot/carbon`
    to get "versions"
    Use `echo phpversion();`
    to get PHP version.

    Some issues can depends on your context, settings,
    macros, timezone, language. So to be sure the code
    you give is enough to reproduce your bug, try it
    first in:
    https://try-carbon.herokuapp.com/?theme=xcode&export&embed

    You can use the [Options] button to change the version
    then when you get the bug with this editor, you can use
    the [Export] button, copy the link of the opened tab,
    then paste it in the issue. Then we can immediatly get
    your issue.
-->


I expected to get:

PHPStan results

<!--
    Always give your expectations. Each use has their owns.
    You may want daylight saving time to be taken into account,
    someone else want it to be ignored. You may want timezone
    to be used in comparisons, someone else may not, etc.
-->

But I actually get:

Config file /var/www/symfony/vendor/nesbot/carbon/extension.neon does not exist or isn't readable
make: *** [phpstan] Error 1

<!--
    If you did not succeed to get the same result in
    https://try-carbon.herokuapp.com/ then precise the
    result you get.
-->

---

I see that in 2.51.0, `extension.neon` was added to `.gitattributes`, so that it is no longer available after a `composer update`. I don't know if the missing `extension.neon` breaks other tools, but it definitely breaks PHPStan.

I originally reported this as a bug to PHPStan and they said this was something that needed to be changed in this project.


Thanks for a great tool!

--- a/.gitattributes
+++ b/.gitattributes
@@ -7,7 +7,6 @@
 /.travis.yml             export-ignore
 /build.php               export-ignore
 /contributing.md         export-ignore
-/extension.neon          export-ignore
 /codecov.yml             export-ignore
 /phpdoc.php              export-ignore
 /phpmd.xml               export-ignore

